pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -564,20 +565,21 @@ def cmd_connection_show(context, name, options):
     # and that name is not current, use it. If current name is same as
     # name, use the current version.
```

Comment:
If no name was specified and there is no current connection but the connections file exists, we get here but name is still not set. So the comment 'Have a name.'  is incorrect, and the logic likely too.